diff -Naur zstd-1.4.5/programs/zstdcli.c zstd-1.4.5.tpg/programs/zstdcli.c
--- zstd-1.4.5/programs/zstdcli.c	2020-05-22 05:04:00.000000000 +0000
+++ zstd-1.4.5.tpg/programs/zstdcli.c	2020-05-22 08:33:46.031001998 +0000
@@ -695,7 +695,7 @@
     if ((filenames==NULL) || (file_of_names==NULL)) { DISPLAY("zstd: allocation error \n"); exit(1); }
     programName = lastNameFromPath(programName);
 #ifdef ZSTD_MULTITHREAD
-    nbWorkers = 1;
+    nbWorkers = 0;
 #endif
 
     /* preset behaviors */
