diff -Naur zstd-1.3.8/programs/zstdcli.c zstd-1.3.8/programs/new-zstdcli.c
--- zstd-1.3.8/programs/zstdcli.c   2018-12-27 13:42:44.000000000 +0100
+++ zstd-1.3.8/programs/new-zstdcli.c       2019-01-01 04:30:50.000000000 +0100
@@ -576,7 +576,7 @@
     cLevel = init_cLevel();
     programName = lastNameFromPath(programName);
 #ifdef ZSTD_MULTITHREAD
-    nbWorkers = 1;
+    nbWorkers = 0;
 #endif
 
     /* preset behaviors */

